gno.land/cmd/gnoland/config.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"flag"
+	"reflect"
+
+	"github.com/gnolang/gno/tm2/pkg/commands"
+)
+
+type configCfg struct {
+	configPath string
+}
+
+// newConfigCmd creates the config root command
+func newConfigCmd(io commands.IO) *commands.Command {
+	cmd := commands.NewCommand(
+		commands.Metadata{
+			Name:       "config",
+			ShortUsage: "config <subcommand> [flags]",
+			ShortHelp:  "Gno config manipulation suite",
+			LongHelp:   "Gno config manipulation suite, for editing base and module configurations",
+		},
+		commands.NewEmptyConfig(),
+		commands.HelpExec,
+	)
+
+	cmd.AddSubCommands(
+		newConfigInitCmd(io),
+		newConfigSetCmd(io),
+		newConfigGetCmd(io),
+	)
+
+	return cmd
+}
+
+func (c *configCfg) RegisterFlags(fs *flag.FlagSet) {
+	fs.StringVar(
+		&c.configPath,
+		"config-path",
+		"./config.toml",
+		"the path for the config.toml",
+	)
+}
+
+// getFieldAtPath fetches the given field from the given path
+func getFieldAtPath(currentValue reflect.Value, path []string) (*reflect.Value, error) {
+	// Look at the current section, and figure out if
+	// it's a part of the current struct
+	field := currentValue.FieldByName(path[0])
+	if !field.IsValid() {
```

Comment:
```suggestion
	if !field.IsValid() || !field.CanSet() {
```

so we don't allow getting unexported fields.

Reply:
Good catch 💯 

[e7be015](https://github.com/gnolang/gno/pull/1605/commits/e7be0150f8901f539b2a82042e8613184be284cc)